ddl/ddl_api.go

```diff
@@ -1153,7 +1192,11 @@ func (d *ddl) AlterTable(ctx sessionctx.Context, ident ast.Ident, specs []*ast.A
 				case ast.TableOptionComment:
 					spec.Comment = opt.StrValue
 					err = d.AlterTableComment(ctx, ident, spec)
+				case ast.TableOptionCharset, ast.TableOptionCollate:
+					toCharset, toCollate := getCharsetAndCollateInTableOption(i, spec.Options)
```

Comment:
This code is weird to me.
Why not return a charset/collate array in `getCharsetAndCollateInTableOption`, and handle the array in a loop calling `d.AlterTableCharsetAndCollate`?

Reply:
addressed.